tests/test_resources/metal_nxo_features.json

```diff
@@ -0,0 +1,132 @@
+{
+  "identifier": {
+    "0": null,
+    "1": null,
+    "2": null,
+    "3": null,
+    "4": null,
+    "5": null,
+    "6": null,
+    "7": null
```

Comment:
identifier should not be null! Oops. I'll fix this upstream and consider whether we should protect against this here.